supplementary_style_guide/glossary_terms_conventions/general_conventions/h.adoc

```diff
@@ -308,41 +308,41 @@ Use "host" to refer to hosts in general, not "hypervisor", "hypervisor host", or
 [discrete]
 [[hyper-threading]]
 ==== image:images/yes.png[yes] Hyper-Threading (noun)
-*Description*: "Hyper-Threading" is the Intel implementation of simultaneous multithreading. If you are not referring specifically to the Intel implementation, use "simultaneous multithreading" or "SMT".
+*Description*: _Hyper-Threading_ is the Intel implementation of simultaneous multithreading. If you are not referring specifically to the Intel implementation, use "simultaneous multithreading" or "SMT".
 
 *Use it*: yes
 
 *Incorrect forms*: hyperthreading, hyper-threading
 
 *See also*:
 
-// Azure: Added "In the Microsoft Windows operating system"
 [discrete]
-[[hyperv]]
-==== image:images/yes.png[yes] Hyper-V (noun)
-*Description*: In the Microsoft Windows operating system, "Hyper-V" is a native hypervisor. Hyper-V can create virtual machines (VMs) on AMD64 systems running the Microsoft Windows operating system. Hyper-V drivers are required on all Red Hat Enterprise Linux (RHEL) VMs running in Microsoft Azure.
+[[hyperconverged]]
+==== image:images/yes.png[yes] hyperconverged (adjective)
```

Comment:
Oh, these changes are a tad bit out of scope? How do I validate these?

Reply:
I didn't change any content. I mistakenly swapped the order of two entries. It's now reverted.